src/dropdown.test.interactions.filterable.ts

```diff
@@ -899,132 +879,161 @@ it('uses `placeholder` as a placeholder when multiselect and no option is select
   expect(input?.placeholder).to.equal('Placeholder');
 });
 
-it('sets `aria-activedescendant` on option hover', async () => {
+it('sets an option as active on hover', async () => {
```

Comment:
I generalized this and similar test names because I added additional assertions (below).